2017/JULY/JULY17/NITIKA/NITIKA.py

```diff
@@ -0,0 +1,17 @@
+//made by DipanshKhandelwal
+if __name__ == '__main__':
```

Comment:
Add the whole code in a `def main()` in which will be called in `if __name__ == '__main__':`
See example codes in repo.